specification/offazurespringboot/resource-manager/Microsoft.OffAzureSpringBoot/preview/2023-01-01-preview/springbootdiscovery.json

```diff
@@ -1550,10 +1650,6 @@
           },
           "description": "The machine ARM id list the app belongs to."
         },
-        "siteName": {
-          "type": "string",
-          "description": "The site name."
-        },
         "springBootVersion": {
           "type": "string",
           "description": "The spring boot version."
```

Comment:
>readOnly [](http://example.com/codeflow?start=11&length=8)

Breaking change.

Reply:
The property name shouldn't exist in the path with the same name, this rule of validation is added recently, so we remove the unused siteName

Reply:
That's fine, but removing a property requires breaking change review.